app/views/devise/passwords/edit.html.erb

```diff
@@ -19,29 +19,17 @@
       <%= render "devise/shared/error_messages", resource: resource %>
       <%= hidden_field_tag "user[reset_password_token]", params[:reset_password_token] %>
 
```

Comment:
Consider: adding a email/username field here to update password managers.
I guessed the `resource_for_reset&.email` would be available here, based on [this](https://github.com/alphagov/govuk-account-manager-prototype/blob/b990613bd7cd105789f735028e07763c10bede78/app/controllers/devise_passwords_controller.rb#L7), but that's likely wrong
```suggestion
      <%= email_field_tag :"user[email]", resource_for_reset&.email, class: "govuk-!-display-none", autocomplete: "off", "aria-hidden": true %>
```